### PR TITLE
created a presence for serendiasquad.com

### PR DESCRIPTION
**Pull request by baybeku**
We share discord bot codes on our site

Viewing a code
4wovybB

Looking a profile
38gCTxV

Editing a code
89ZUUhf

Received an error
5FudYQT

### CODE REVIEW

File: websites/S/Serendia Squad/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    "name": "BBQ",
+    "id": "622527559946731540"
+  },
+  "url": "serendiasquad.com",
+  "description": {
+    "tr": "The Serendia Squad's code sharing website.",
+	"az": "Serendia Squad'ın kod paylaşım saytı.",
+    "en": "Serendia Squad'ın kod paylaşım sitesi.",
+	"fr": "Site de partage de code de Serendia Squad's."
+  },
```
Comment (Bas950) [inline websites/S/Serendia Squad/dist/metadata.json:13]: i think you kinda facked up here....

Comment (NMW03) [reply websites/S/Serendia Squad/dist/metadata.json:13]: azerbaijani description is wrong

Comment (doomlerd) [reply websites/S/Serendia Squad/dist/metadata.json:13]: they all are

Comment (baybeku) [reply websites/S/Serendia Squad/dist/metadata.json]: > they all are

sorry its my mistake i uploaded old files 😂

File: websites/S/Serendia Squad/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  "url": "serendiasquad.com",
+  "description": {
+    "tr": "Serendia Squad'ın kod paylaşım sitesi.",
+    "en": "Serendia Squad's code sharing website.",
+  },
+  "service": "Serendia Squad",
+  "version": "1.0.0",
+  "logo": "https://bbq.is-pretty.cool/5o6DS5b.png",
+  "thumbnail": "https://bbq.is-pretty.cool/5o6DS5b.png",
```
Comment (frozaum) [inline websites/S/Serendia Squad/dist/metadata.json:15]: Please change the thumbnail to a promotional image or a screenshot of the website

File: websites/S/Serendia Squad/dist/metadata.json [hunk 3]
```diff
@@ -12,7 +12,7 @@
   "service": "Serendia Squad",
   "version": "1.0.0",
   "logo": "https://bbq.is-pretty.cool/5o6DS5b.png",
-  "thumbnail": "https://bbq.is-pretty.cool/5o6DS5b.png",
+  "thumbnail": "https://barbecue.is-serious.business/vtX7xgA.png",
```
Comment (Bas950) [inline websites/S/Serendia Squad/dist/metadata.json]: upload to imgur

Comment (frozaum) [reply websites/S/Serendia Squad/dist/metadata.json]: bro it's owo

Comment (frozaum) [reply websites/S/Serendia Squad/dist/metadata.json]: bruh this is [owo](https://whats-th.is/)

**Pull request merged**